Makefile

```diff
@@ -280,7 +280,13 @@ jsformat:
 .PHONY: jstest
 # Run JS unit tests.
 jstest:
+ifndef CIRCLECI
 	cd frontend; yarn run test
+else
+	# Use --runInBand on CircleCI, per
+	# https://jestjs.io/docs/en/troubleshooting#tests-are-extremely-slow-on-docker-andor-continuous-integration-ci-server
+	cd frontend; yarn run test --runInBand
+endif
```

Comment:
do we need to say `endif #CIRCLECI` like we do for `jslint:`?

Reply:
No, that's just an additional comment for the user, since the matching if clause is much further away